nemo_skills/inference/eval/swebench.py

```diff
@@ -182,6 +183,105 @@ def __init__(self, cfg: SweBenchGenerationConfig):
 
         self.should_run_evaluation = False
         self.evaluator = None
+        self._reasoning_warning_shown = False
```

Comment:
does it not make sense to run `super().__init__(cfg)` here instead? This would cut down on some of the other boilerplate above too.